Fix components requirements to make pkg_resources work correctly.

This was already fixed in the upstream:
https://github.com/cherrypy/cheroot/commit/18447cbee7135f3f5017c8f6a9607dc1f7259570

--- cheroot-6.3.2/setup.py
+++ cheroot-6.3.2/setup.py
@@ -45,7 +45,7 @@ params = dict(
     ),
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
     install_requires=[
-        'backports.functools_lru_cache',
+        'backports.functools_lru_cache; python_version<"3.3"',
         'six>=1.11.0',
         'more_itertools>=2.6',
     ],
